README.md

```diff
@@ -28,7 +28,7 @@ A list of companies that have paid Developer Community Writer Programs.
 - [Dev Spotlight](https://www.devspotlight.com/jobs/) - Around $300-$500 per piece depending on length and content
   > Technical content production agency that works with many clients.
 
-- [Digital Ocean](https://www.digitalocean.com/write-for-donations/) - Up to $400 per piece
+- [Digital Ocean](https://www.digitalocean.com/write-for-donations/) - Up to $400 per piece ****Currently paused accept new writer**
```

Comment:
Hey @krissnawat thanks for the addition. Could you change the text to 

`** *Applications currently on hold**`